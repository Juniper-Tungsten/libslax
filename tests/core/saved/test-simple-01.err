compilation error: file ../../../tests/core/test-simple-01.slax line 65 element number
xsl:number : letter-value 'alphabetic' not implemented
Unimplemented block at ../../libxslt/preproc.c:1476
before
This is 'one' is  good
