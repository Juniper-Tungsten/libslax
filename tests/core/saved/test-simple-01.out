<!DOCTYPE html PUBLIC "string" "string">
<doc xmlns:axsl="http://www.w3.org/1999/XSL/Transform"><mumble><two>002</two><three>0</three></mumble><doc test10="onehand" test12="anotherhand" hand="This is 'one'">heaven
  fish
  This is 'one'
  This is 'two'
  10
  12
  hand
  yup
  fish
  trout
  this thing
  that thing
  10000

  
    phil
    45
    It worked!!
  

  
    raleigh
    nc
    27614
    1312
  

  7.1R1.4

  yes

</doc><name></name><one></one><two></two><a></a><b></b><thing></thing><test></test><fish></fish><trout></trout><this></this><that></that><mtu></mtu><results></results><location></location><version></version><_name></_name></doc>
